pkg/cmd/server/origin/master_config.go

```diff
@@ -827,8 +827,9 @@ func newAuthorizationAttributeBuilder(requestContextMapper kapi.RequestContextMa
 		requestInfoFactory,
 	)
 	personalSARRequestInfoResolver := authorizer.NewPersonalSARRequestInfoResolver(browserSafeRequestInfoResolver)
+	projectRequestInfoResolver := authorizer.NewProjectRequestInfoResolver(personalSARRequestInfoResolver)
```

Comment:
> final := chainFunc(
>     browserSafeRequestInfoResolver,  // base info
>     authorizer.NewPersonalSARRequestInfoResolver,  // wrapper funcs
>     authorizer.NewProjectRequestInfoResolver,
> )

No real disagreement, but I suspect this gets reswizzled in the rebase.  You want the related updates now?